packages/pdf-viewer/pdfSource.test.ts

```diff
@@ -0,0 +1,41 @@
+import { PdfData } from './pdfSource';
+import * as pdfjsLib from 'pdfjs-dist';
+
+const path = require('path');
+const fs = require('fs');
```

Comment:
Don't import everything, instead import only what you need. `import { readFile} from 'fs'` Same for path.